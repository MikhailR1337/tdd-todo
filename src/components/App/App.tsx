import React from 'react';
import './App.scss';

export function App() {
    return (
        <>
            <div className="App">
                Hello World!
            </div>
            <ul>
                <li>
                    Покушать
                </li>
                <li>
                    Погулять
                </li>
                <li>
                    Поесть
                </li>
            </ul>
        </>
    );
}
