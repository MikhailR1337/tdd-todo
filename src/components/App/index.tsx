import React from 'react';
import 'materialize-css/dist/css/materialize.min.css';
import { Header } from '../Header';
import { AddTodo } from '../AddTodo';

export function App() {
    return (
        <div className='row'>
            <Header />
            <div className='container center-align'>
                <AddTodo />
            </div>
        </div>
    );
}
