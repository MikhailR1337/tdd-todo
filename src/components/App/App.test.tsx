import React from 'react';
import { render, screen } from '@testing-library/react';
import { App } from './App'

test('App has Hello World', () => {
  render(<App />);
  expect(screen.getByText(/Hello World/i)).toBeInTheDocument();
});

test('App has List', () => {
  render(<App />);
  expect(screen.getByRole('list')).toBeInTheDocument();
});
