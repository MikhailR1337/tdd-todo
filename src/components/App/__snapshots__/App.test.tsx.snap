// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple App tests App snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row"
      >
        <nav
          class="center-align"
        >
          <div
            class="nav-wrapper"
          >
            I am mister Header
          </div>
        </nav>
        <div
          class="container center-align"
        >
          <div
            class="row valign-wrapper"
          >
            <div
              class="input-field col s6"
            >
              <input
                id="input_text"
                type="text"
                value=""
              />
            </div>
            <div
              class="col s6"
            >
              <a
                class="waves-effect waves-light btn"
              >
                add
              </a>
            </div>
          </div>
        </div>
        <div
          class="container center-align"
        >
          <ul
            class="collapsible"
          >
            <li>
              <div
                class="collapsible-header"
              >
                <i
                  class="material-icons"
                >
                  free_breakfast
                </i>
                watching shkya
                <span
                  class="badge"
                />
                <a
                  class="waves-effect waves-light btn"
                >
                  remove
                </a>
              </div>
            </li>
            <li>
              <div
                class="collapsible-header"
              >
                <i
                  class="material-icons"
                >
                  free_breakfast
                </i>
                BBQ on macbook with docker
                <span
                  class="badge"
                />
                <a
                  class="waves-effect waves-light btn"
                >
                  remove
                </a>
              </div>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row"
    >
      <nav
        class="center-align"
      >
        <div
          class="nav-wrapper"
        >
          I am mister Header
        </div>
      </nav>
      <div
        class="container center-align"
      >
        <div
          class="row valign-wrapper"
        >
          <div
            class="input-field col s6"
          >
            <input
              id="input_text"
              type="text"
              value=""
            />
          </div>
          <div
            class="col s6"
          >
            <a
              class="waves-effect waves-light btn"
            >
              add
            </a>
          </div>
        </div>
      </div>
      <div
        class="container center-align"
      >
        <ul
          class="collapsible"
        >
          <li>
            <div
              class="collapsible-header"
            >
              <i
                class="material-icons"
              >
                free_breakfast
              </i>
              watching shkya
              <span
                class="badge"
              />
              <a
                class="waves-effect waves-light btn"
              >
                remove
              </a>
            </div>
          </li>
          <li>
            <div
              class="collapsible-header"
            >
              <i
                class="material-icons"
              >
                free_breakfast
              </i>
              BBQ on macbook with docker
              <span
                class="badge"
              />
              <a
                class="waves-effect waves-light btn"
              >
                remove
              </a>
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
